cips/cip-6-second-blocks.md

```diff
@@ -0,0 +1,60 @@
+| cip | TBD |
+| - | - |
+| title | Reduce block time to six seconds |
+| description | Proposal to reduce block time on celestia-app to 6 seconds, from 12. |
+| author | Josh Stein ([@jcstein](https://github.com/jcstein)) |
+| discussions-to | <https://forum.celestia.org/t/cip-decrease-block-time-to-6-seconds/1836> |
+| status | Draft |
+| type | Standards Track |
+| category | Core |
+| created | 2024-10-09 |
+
+## Abstract
+
+This CIP proposes to reduce the block time on celestia-app to 6 seconds, from 12. This will increase the throughput by double and and reduce the time it takes for transactions to be finalized by half.
```

Comment:
[very optional] grammar stuff

```suggestion
This CIP proposes to reduce the block time on celestia-app to 6 seconds, from 12. This will double the throughput and reduce the time it takes for transactions to be finalized in half.
```